src/PHPCR/Util/QOM/Sql2ToQomQueryConverter.php

```diff
@@ -830,6 +830,53 @@ protected function parseLiteralValue()
     }
 
     /**
+     * 6.7.34 Fulltext Literal
+     * Parse an SQL2 literal value
+     *
+     * @return LiteralInterface
+     */
+    protected function parseFulltextLiteral()
```

Comment:
I think this method was accidentally added in https://github.com/phpcr/phpcr-utils/commit/57dbaef8d84920e940e03dd8dab1c1d5c92650e6 and removed in the following commit https://github.com/phpcr/phpcr-utils/commit/72fc31f88120cf9b91ed1b9c19c2ab0c9183b693